lib/vanity/frameworks/rails.rb

```diff
@@ -53,6 +53,17 @@ def use_vanity(symbol = nil, &block)
           define_method :vanity_identity do
             return @vanity_identity if @vanity_identity
 
+            cookie = lambda do |value|
+              result = { value: value, 
```

Comment:
Style nitpick (sorry), can you put the hash entries on their own lines, e.g.:
```
result = {
  value: value,
  ...
}
```

and make sure there's no trailing whitespace?